docs/enroute/Melbourne Centre/ASP.md

```diff
@@ -76,5 +78,32 @@ Any aircraft that will enter AS ADC airspace, and not landing at YBAS, must be H
     <span class="hotline">**ASP** -> **AS ADC**</span>: "via AS, JDS"  
     <span class="hotline">**AS ADC** -> **ASP**</span>: "JDS, A060"
 
+### WR ADC
+
+Departures from YPWR in to WRA Class E will be coordinated when ready for departure.
```

Comment:
Specifically CTA, Not Class E. A VFR aircraft going in to WRA Class E need not be coordinated.